api/apps/api/src/modules/scenarios/locks/scenario.lock.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { CreateDateColumn, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity(`scenario_locks`)
+export class ScenarioLockEntity {
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `user_id`,
+  })
+  userId!: string;
+
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `scenario_id`,
+  })
+  scenarioId!: string;
+
+  @CreateDateColumn({ name: 'created_at', type: 'timestamp' })
```

Comment:
possibly could have `NOT NULL default now()`

Reply:
Sorry Kamil, I didn't understand this comment: I added `NOT NULL default now()` in the migration that creates the column - do you mean we should add that here as well?

Reply:
@ikas I believe so - in the end, we should aim at restoring the possibility to run migration generator - but sure this requires much more work (so adding it there would just make less effort later). Up2 you : )

Reply:
you mean like this? https://github.com/Vizzuality/marxan-cloud/pull/673/commits/eba3a61e200d29e9c3bca93accda678ade925e3e

Reply:
@ikas ✅ 👍